include/seqan/modifier/modifier_shortcuts.h

```diff
@@ -292,7 +292,8 @@ template < typename TSequence, typename TSpec, typename TParallelTag >
 inline void reverseComplement(StringSet<TSequence, TSpec> & stringSet, Tag<TParallelTag>)
 {
     int seqCount = length(stringSet);
-    SEQAN_OMP_PRAGMA(parallel for if(IsSameType<Tag<TParallelTag>, Parallel>::VALUE))
+    SEQAN_OMP_PRAGMA(parallel for if(IsSameType<Tag<TParallelTag>, Parallel>::VALUE &&
```

Comment:
Can you add a short message why we exclude `isStringSetConcatDirect` and `isStringPacked`?